lib/Perl/Critic/Policy/ControlStructures/ProhibitYadaOperator.pm

```diff
@@ -0,0 +1,95 @@
+package Perl::Critic::Policy::ControlStructures::ProhibitYadaOperator;
+
+use 5.006001;
+use strict;
+use warnings;
+use Readonly;
+
+use Perl::Critic::Utils qw{ :characters :severities };
+use base 'Perl::Critic::Policy';
+
+our $VERSION = '1.123';
+
+#-----------------------------------------------------------------------------
+
+Readonly::Scalar my $DESC => q{yadda operator (...) used};
+Readonly::Scalar my $EXPL => [ 100 ];
+
+#-----------------------------------------------------------------------------
+
+sub supported_parameters { return ()                         }
+sub default_severity     { return $SEVERITY_HIGH             }
+sub default_themes       { return qw( core pbp maintenance ) }
+sub applies_to           { return 'PPI::Token::Operator' }
+
+#-----------------------------------------------------------------------------
+
+sub violates {
+    my ( $self, $elem, undef ) = @_;
+
+    if ( _is_yada( $elem ) ) {
+        return $self->violation( $DESC, $EXPL, $elem );
+    }
+    return;    #ok!
+}
+
+sub _is_yada {
+    my ( $elem ) = @_;
+
+    return unless $elem eq '...';
+    return unless $elem->statement;
+    return if ref($elem->snext_sibling) eq 'PPI::Token::Number' and ref($elem->sprevious_sibling) eq 'PPI::Token::Number';
```

Comment:
I think just about anything (not just numbers) could be a valid operand for `...`. It might be better to check if *any* operands exist. If so, then it's probably not a yada operator.